packages/build/src/plugins_core/deploy/build_service.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+option go_package = "github.com/netlify/buildbot/buildservice/proto";
+
+package buildservice;
+
+// Service to communicate between Netlify Build and the buildbot
+service BuildService {
+  // Request the buildbot to deploy the site
+  rpc RunPostDeploy(PostDeployRequest) returns (PostDeployResponse) {}
+}
+
+message PostDeployRequest {
+  // Whether the request should wait for the deploy (including post-processing)
+  // to finish before completing
+  bool await_live = 1;
+}
+
+message PostDeployResponse {
+  // If false, `error` and `error_type` should be checked
+  bool succeeded = 1;
+  // Error message
+  string error = 2;
+  // Either 'user' or 'system'
+  string error_type = 3;
```

Comment:
Should we use an `enum` for this? See https://developers.google.com/protocol-buffers/docs/proto3#enum

Reply:
I kept it as a string to simplify the implementation (since it does not require importing the enum type in the code).
Also, we currently use a string in the buildbot, so this minimizes the amount of work in the buildbot. Finally, this attribute will be removed altogether in the next step of this feature (when Netlify Build becomes the server instead of the client).

However, both `enum` and `string` work. @vbrown608 Which one do you prefer?

Reply:
Since this is going to be removed in the next step feel free to disregard my comment.